data/template/buf.connect-go.gen.yaml

```diff
@@ -0,0 +1,13 @@
+# This template is to generate connect go.
+version: v1
+managed:
+  enabled: true
+  go_package_prefix:
+    default: github.com/bufbuild/buf/private/gen/proto/go
+    override:
```

Comment:
I don't think we need this `override` since we're in the repo with the generated code?

Reply:
Ah yeah good call.  Removed.